config/locales/mailers/en.yml

```diff
@@ -50,6 +50,25 @@ en:
         part1: "Your work \"%{title}\" was deleted from the Archive by a site admin."
         part2: "If your work was part of an import project managed by our Open Doors team, please contact Open Doors (%{opendoors_link}) with any further questions."
         part3: "If it's possible your work violated the Archive's Terms of Service, please contact our Abuse Committee (%{abuse_link})."
+    admin_hidden_work_notification:
+      subject: "[%{app_name}] Your work has been hidden by the Abuse Team"
+      hello: "Dear %{name},"
+      tos: "Terms of Service"
+      abuse_committee: "contact Policy & Abuse"
+      sincerely: "Sincerely,"
+      sender: "AO3 Abuse"
+      html:
+        part1: "Your work %{title} has been hidden by the Policy & Abuse team and is no longer publicly accessible."
+        part2: "While your work is hidden, you will still be able to access it through the link provided above, but it will not be listed on your works page, and it won't be available to other users of the Archive."
+        part3: "Please check your email, including your spam folder, as the Policy & Abuse team may have already contacted you explaining why your work was hidden."
+        part4: "If your work was hidden due to being in violation of the Archive of Our Own's %{tos_link}, you will be required to take action to correct the violation. Failure to bring your work into compliance with the Terms of Service may lead to your work being deleted from the Archive."
+        part5: "If you are uncertain why your work was hidden, and you have not received further communication regarding this matter, please %{abuse_link} directly."
+      text:
+        part1: "Your work \%{title}\ (%{work_url}) has been hidden by the Policy & Abuse team and is no longer publicly accessible."
+        part2: "While your work is hidden, you will still be able to access it through the link provided above, but it will not be listed on your works page, and it won't be available to other users of the Archive."
+        part3: "Please check your email, including your spam folder, as the Policy & Abuse team may have already contacted you explaining why your work was hidden."
+        part4: "If your work was hidden due to being in violation of the Archive of Our Own's Terms of Service (%{tos_link}), you will be required to take action to correct the violation. Failure to bring your work into compliance with the Terms of Service may lead to your work being deleted from the Archive."
+        part5: "If you are uncertain why your work was hidden, and you have not received further communication regarding this matter, please contact Policy & Abuse directly: (%{abuse_link})"
```

Comment:
Oops, my bad: there shouldn't be parentheses around the URL here, although there should be a period at the end.